sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClient.java

```diff
@@ -98,22 +102,42 @@
      * Starts the search for available phone numbers to purchase.
      *
      * @param countryCode The ISO 3166-2 country code.
-     * @param searchRequest {@link PhoneNumberSearchRequest} specifying the search request
-     * until it gets a result from the server
+     * @param phoneNumberType {@link PhoneNumberType} The phone number type
+     * @param assignmentType {@link PhoneNumberAssignmentType} The phone assignment type
```

Comment:
```suggestion
     * @param assignmentType {@link PhoneNumberAssignmentType} The phone number assignment type.
```
Nit: let's be consistent with full stops at the end.